src/components/Navigation/Navigation.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { Link } from 'gatsby';
+import { Helmet } from 'react-helmet';
+import PropTypes from 'prop-types';
+import { slide as Menu } from 'react-burger-menu';
+import favicon from '../../images/favicon.ico';
+import '../../stylesheets/components/Navigation.scss';
+
+const Navigation = ({ data, children }) => {
```

Comment:
I'm not sure if this makes sense as a single component, at least not with the name "Navigation". Normally, if I saw a component by the name of Navigation, I would think that it would only contain an independent navigation. However, what you really have here is something that defines the entire application layout.

For example, where you actually use this component, you have:

```jsx
  <Navigation>
    <Home />
  </Navigation>
```

But this looks kind of weird, because from reading it, it looks like you're putting the Home page _inside_ of the navigation. But that's not what's actually happening; what you really have is the Home page rendered inside of the main application layout. The navigation is just a single section of the page that links to other pages; so really, what you have named `Navigation` is not a navigation, but a page layout that includes three separate navigation components within it.

What I would recommend is moving this `Navigation` component into your `src/pages/index.js` file, and just having it _be_ that page's component, without assigning it a name.

I would leave the other three navigation components in this file, but instead of having a default export, export the three components so that you can use them in the `src/pages/index.js` page.

If in the future we want to define multiple page layouts, then we can create a collection of layout components that may each independently reuse some or all of the navigation components.

Reply:
Wont this clutter my src/pages/index.js? What if I rename to Navigation to Layout? Would that make more sense?